src/compiler/types.ts

```diff
@@ -1807,8 +1807,8 @@ namespace ts {
 
     export interface CatchClause extends Node {
         kind: SyntaxKind.CatchClause;
-        parent?: TryStatement;
-        variableDeclaration: VariableDeclaration;
+        parent?: TryStatement; // We parse missing try statements

```

Comment:
What does this comment mean? This property is present to reduce costs when walking up the AST.